ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1415,6 +1415,39 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe
     workflow: rosa-aws-sts-hypershift
+- as: aws-rosa-sts-hypershift-guest-int-priv-link-full-f2
```

Comment:
Let us use the full word `private` instead of `priv`

Reply:
I have to use the priv to avoid exceeding the length limit of job name (61bytes)

Reply:
Just checked again using `int` for `integration` we could use the full word `private `